pkg/constants/constants.go

```diff
@@ -39,6 +39,19 @@ const (
 	// ConfigFilePath is the absolute path to the NATS config file.
 	ConfigFilePath = ConfigMapMountPath + "/" + ConfigFileName
 
+	// PidFileVolumeName is the name of the volume used for the NATS server pid file.
+	PidFileVolumeName = "pid"
+
+	// PidFileName is the pid file name.
+	PidFileName = "gnatsd.pid"
+
+	// PidFileMountPath is the absolute path to the directory where NATS
+	// will be leaving its pid file.
+	PidFileMountPath = "/var/run/nats"
+
+	// PidFilePath is the location of the pid file.
+	PidFilePath = PidFileMountPath + "/" + PidFileName
```

Comment:
Would it be simpler for users to just set the `PidFilePath` and you get the `PidFileMountPath` by parsing?  I don't see `PidFileName` used anyplace else.  wdyt?

Reply:
These aren't tunable for users so think ok just as constants for now